### DISCUSSION TITLE
Trie Search

### DISCUSSION THREAD

**Discussion by gabriags** [harpomaxx/algoritmos2#9] [General]
Una consulta con respecto a Search de Trie. ¿Consideramos palabras contenidas en otras pero que no son elementos como un caso aparte? 

![Captura](https://user-images.githubusercontent.com/126571813/227570795-34955288-1aa5-42df-94f7-a56ce35fa340.PNG)

**Comment by harpomaxx** [comment]
Se consideran las palabras completas. Los  "elementos' no se consideran al menos en esta implementación de Trie.
